### ISSUE TITLE
Raw Images don't support Specials

### ISSUE THREAD

**Issue by NotxNero** [Kareadita/Kavita#1737]
Please put as much information as possible to help me understand your issue. OS, browser, version are very important!

**Describe the bug**
When adding a Special using the correct folder structure and SP tag in the form of a folder filled with jpg's Kavita shows every jpg as a seperate special. This is not the case when the series is only CBZ files.  

**To Reproduce**
1. Have a manga folder 
2. use the structure ROOT > SERIES FOLDER > SPECIALS FOLDER > SERIES NAME SP01 > JPGs
3. Scan Kavita / Refresh Kavita

**Expected behavior**
Every jpg from the sp01 shows up as a seperate folder

**Screenshots**

![image](https://user-images.githubusercontent.com/122322489/211423690-c418da14-41d0-4a8f-826e-591904a2b466.png)

![image](https://user-images.githubusercontent.com/122322489/211423702-bf3442d8-2a2d-4e84-8134-f03bb50746af.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 0.5.6.0

**Comment by majora2007** [comment]
So I took a look at this and it looks like Kavita doesn't support Specials for Raw Images. This is because each image is treated as a special already. When I added parsing support to recognize SP marker, the logic takes the name of the folder with the marker and treats that as a series. 

Given the extremely low population of users using raw images (especially how limiting they are), I would opt to not support this and instead ask you to open a feature request. This way if the community deems it important, I can reconsider and dump the many hours it would take to support this.
